etcdctlv3/command/lease_command.go

```diff
@@ -63,8 +62,9 @@ func leaseCreateCommandFunc(cmd *cobra.Command, args []string) {
 		ExitWithError(ExitBadArgs, fmt.Errorf("bad TTL (%v)", err))
 	}
 
-	req := &pb.LeaseCreateRequest{TTL: ttl}
-	resp, err := mustClient(cmd).Lease.LeaseCreate(context.Background(), req)
+	c := mustClient(cmd)
+	l := clientv3.NewLease(c)
```

Comment:
It would be easier to implement `client.Close()` if only clientv3 could call `newLease` and exposed the `Lease` as a field in `clientv3.Client` (same with the kv stuff too). Is there a use case for having multiple outstanding `Leases` per client?